admin/javascript/SSNotice.js

```diff
@@ -0,0 +1,205 @@
+/**
+* File: SSNotice.js
+*
+* Replace the old statusMessage/errorMessage
+* with a custom library: ss.notice
+*
+* Allows for concistent notices/message display throughout the CMS
+* via simple JavaScript function calls:
+* - ss.notice.message(text, [options]);
+* - ss.notice.success(text, [options]);
+* - ss.notice.warning(text, [details], [options]);
+* - ss.notice.error(text, [details], [options]);
+**/
+(function($, root) {
+
+	// local working version of the library object
+	var lib = {};
+
+	/**
+	 * Library settings
+	 */
+	lib.settings = {
+		parentSelector: '.cms-content-fields',
+		$notice: $('<p class="message"/>'),
```

Comment:
Using `options.type` to add a set of custom classes is pretty bad; it seems like a way to shoe-horn style aspects into a element that is being created by JS.

I'm not sure that rendering and inserting a new node into the dom is less manipulation that just inserting text into an existing element; but it's beside the point.

We're working with an MVC framework and that should be core to our approach throughout our project. Here, we're clearly coupling our control with our view. Writing HTML in JS is no different to doing it in PHP and should be avoided as they are separate concerns.